pages/returning-members.js

```diff
@@ -0,0 +1,282 @@
+    "Yes, I will continue on the team remotely only",
+    "Undecided or unsure", 
+    "No, taking the term off" 
+]
+
+const MEMBERSHIP = [
+    'Member', 
+    'Exec/lead/advisor', 
+    'Coop', 
+    'Not active on Waterloop this term',
+];
+
+const CONTINUINGSTATUS = [
+    'Continue with my sub-team', 
+    'Transfer to another sub-team (please specify)', 
+    'Want to take on a leadership role - lead', 
+    'Want to take on a leadership role - co-op supervisor', 
+    'Want to become a co-op', 
+    'I\'m undecided or not continuing',
+]
+
+const FormHeader = ({title, marginBottom}) => {
+    const theme = useContext(ThemeContext);
+    return (
+        <SystemComponent
+            fontSize={theme.fontSizes.header3}
+            textAlign='center'
+            mb={marginBottom}
+        >
+            <SystemSpan>
+                <Subtitle>{title}</Subtitle>
+            </SystemSpan>
+        </SystemComponent>
+    );
+}
+
+const RegistrationForm = () => {
+    const theme = useContext(ThemeContext);
+
+    const [formValues, setFormValues] = useState({
+        fullName:  "",
+        termDescription: "", 
+        previousTerms: [], 
+        futureTerms: [], 
+        subteam: "",
+        activeStatus: "", 
+        continuingStatus: "", 
+        email: "", 
+        comments: "",
+        futureTasks: "", 
+    });
+
+    const [hasError, setHasError] = useState({
+        fullName: false,
+        termDescription: false, 
+        membership: false, 
+        previousTerms: false, 
+        futureTerms: false,
+        activeStatus: false, 
+        subteam: false,
+        activeStatus: false,
+        continuingStatus: false,
+        email: false, 
+    });
+
+    const handleSave = () => {
+        console.log('clicked button')
+        console.log(formValues)
+        const updatedErrorList = {...hasError};
+
+        for (const [name, value] of Object.entries(updatedErrorList)) {
+            updatedErrorList[name] = false;
+        }
+
+        if (!formValues.fullName || formValues.fullName.split(' ').length !== 2) {
+            updatedErrorList['fullName'] = true;
+        }
+        if (!formValues.email || !isEmail(formValues.email)) {
+            updatedErrorList['email'] = true;
+        }
+        if (!formValues.termDescription) {
+            updatedErrorList['termDescription'] = true;
+        }
+        if (!formValues.subteam) {
+            updatedErrorList['subteam'] = true;
+        }
+        if (!formValues.continuingStatus) {
+            updatedErrorList['continuingStatus'] = true;
+        }
+        if (!formValues.activeStatus) {
+            updatedErrorList['activeStatus'] = true;
+        }
+        if (!formValues.futureTerms) {
+            updatedErrorList['futureTerms'] = true;
+        }
+
+        setHasError(updatedErrorList);
+
+        if (Object.values(updatedErrorList).some(hasError => hasError === true)) {
```

Comment:
Personally, this is another lesson in code readability. It could be better to do: 
```
const submissionHasErrors = Object.values(updatedErrorList).some(hasError => hasError === true)
if (submissionHasErrors) {
    return;
}
```

Another way would be to put `Object.values(updatedErrorList).some(hasError => hasError === true)` into a separate function called "submissionHasErrors" for better readability. However, this code snippet here may be too short to warrant putting into a separate function altogether.